src/common/decorators/local-storage.ts

```diff
@@ -32,9 +33,13 @@ class Storage {
 
 const storage = new Storage();
 
-export const LocalStorage = (key?: string) => {
-  return (element: ClassElement, propName: string) => {
-    const storageKey = key || propName;
+export const LocalStorage = (
```

Comment:
Should we listen to the `storage` event? 

https://developer.mozilla.org/en-US/docs/Web/API/Window/storage_event

Reply:
For when it is changed outside of the class? Would be nice, but we don't have a use case now I think?

Reply:
Two windows open. Changes to one are applied to the other

Reply:
Added